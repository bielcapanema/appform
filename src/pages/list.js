export default function list(users) {
  return `<ol>
    ${users.map((user) => {
      return `<li class="card">
      <h4>${user.name}</h4>
      <p><strong>E-mail</strong>: ${user.email}</p>
      <p><strong>CPF</strong>: ${user.cpf}</p>
      <p><strong>Telefone</strong>: ${user.phone}</p>
    </li>`
    }).join('')}
    </ol>`
}