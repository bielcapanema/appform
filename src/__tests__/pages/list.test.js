import list from '../../pages/list'

test('List render', () => {
  expect(list([])).toMatchSnapshot()
})