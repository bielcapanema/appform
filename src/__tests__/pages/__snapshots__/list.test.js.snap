// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List render 1`] = `
"<div class=\\"wrapper\\">
    <h2>App Form</h2>
    <h6>Utilizando VanillaJS</h6>
    <nav class=\\"tabs\\">
      <div class=\\"selector\\"></div>
      <a onClick=\\"window.goTo('/cadastro')\\" class=\\"\\">Cadastro</a>
      <a onClick=\\"window.goTo('/lista')\\"  class=\\"\\">Lista</a>
    </nav>
  </div>
    <div class=\\"list\\">
      <h3>Nenhum usu√°rio cadastrado.</h4>
    </div>"
`;
